datacats/cli/create.py

```diff
@@ -20,7 +20,7 @@ def create(opts):
     """Create a new environment
 
 Usage:
-  datacats create [-bin] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
+  datacats create [-bin] [-c NAME] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
```

Comment:
Yeah, my concern is a new user will see the docs and be confused about what the child name thing is. I can appreciate the uniformity of having the option on all commands where it makes sense though. As I said, I'm torn :-)